src/components/Section/Identification/ContactInformation/ContactInformation.test.jsx

```diff
@@ -58,11 +58,15 @@ describe('The ContactInformation component', () => {
       ]
     }
     const component = mount(<ContactInformation {...expected} />)
-    expect(component.find('.table-cell strong').length).toEqual(5)
-    expect(component.find('.table-cell strong').at(0).text()).toEqual('(202) 867-5309 x1234')
-    expect(component.find('.table-cell strong').at(1).text()).toEqual('(202) 867-5309')
-    expect(component.find('.table-cell strong').at(2).text()).toEqual('867-5309')
-    expect(component.find('.table-cell strong').at(3).text()).toEqual('+001 1234567890 x1234')
-    expect(component.find('.table-cell strong').at(4).text()).toEqual('+001 1234567890')
+    // NOTE: This was commented out due to prototyping for next usability testing.
+    // Please removed or uncomment based on results.
+    // expect(component.find('.table-cell strong').length).toEqual(5)
```

Comment:
Just curious, is there list of places where code was changes specifically for user testing?  If not, keeping one somewhere might make it easy to roll the changes back if the user testing suggests that's the right move.  Just a thought.